src/storage/secureStorage.ts

```diff
@@ -50,13 +52,30 @@ export const createSecureAccount = async ({
   } else {
     mnemonic = new Mnemonic(givenMnemonic)
   }
+    
+  if (vanity) {
+    let done = false
+    while (!done){
+      let keypair, address 
+      const { keypair: kp, address: addr } = await Keypair.fromMnemonic(mnemonic, netType)
+      if (addr.b58.includes(vanity)){
```

Comment:
You probably want to do .startsWith?

Also this is using helium address but you probably care more about the resulting solana address.